src/main/menu.ts

```diff
@@ -221,4 +221,38 @@ export function buildMenu(windowManager: WindowManager) {
 
   const menu = Menu.buildFromTemplate(Object.values(appMenu));
   Menu.setApplicationMenu(menu);
+
+  if (isTestEnv) {
+    // this is a workaround for the test environment (spectron) not being able to directly access
+    // the application menus (https://github.com/electron-userland/spectron/issues/21)
```

Comment:
Having read that issue, is there a reason why you didn't go with some of the addon packages that are mentioned in that issue?

Reply:
Yes, from reading the comments I didn't have confidence that they work as is and are supported